From 37bb50603a05c16767b3020c0f7b5be030f2f160 Mon Sep 17 00:00:00 2001
From: Ahmed Ismail <Ahmed.Ismail@arm.com>
Date: Wed, 27 Dec 2023 15:15:05 +0000
Subject: [PATCH] asm-syntax: Replace incorrect type specifier

In C++ files we should use __asm rather than __ASM as __ASM
is not known as a correct C++ type specifier.

Signed-off-by: Ahmed Ismail <Ahmed.Ismail@arm.com>
---
 source/application/main/Main.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/application/main/Main.cc b/source/application/main/Main.cc
index bbe35d9..d36d54f 100644
--- a/source/application/main/Main.cc
+++ b/source/application/main/Main.cc
@@ -28,7 +28,7 @@
 extern void main_loop();
 
 #if defined(__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050)
-__ASM(" .global __ARM_use_no_argv\n");
+__asm(" .global __ARM_use_no_argv\n");
 #endif
 
 /* Print application information. */
-- 
2.34.1

